plugins/tokenizers/mecab.c

```diff
@@ -1063,6 +1064,34 @@ mecab_next_default_format(grn_ctx *ctx,
   }
 }
 
+int
```

Comment:
We must use `static` for internal function:

```suggestion
static int
```